backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs

```diff
@@ -10,12 +12,39 @@ public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.PimsSecurityDeposit, Model.SecurityDepositModel>()
                 .Map(dest => dest.Id, src => src.SecurityDepositId)
-                .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
-                .Map(dest => dest.SecurityDepositTypeId, src => src.SecurityDepositTypeCode)
-                .Map(dest => dest.SecurityDepositType, src => src.SecurityDepositTypeCodeNavigation.Description)
                 .Map(dest => dest.Description, src => src.Description)
                 .Map(dest => dest.AmountPaid, src => src.AmountPaid)
-                .Map(dest => dest.DepositDate, src => src.DepositDate);
+                .Map(dest => dest.DepositDate, src => src.DepositDate)
+                .Map(dest => dest.DepositType, src => src.SecurityDepositTypeCodeNavigation)
+                .Map(dest => dest.OtherTypeDescription, src => src.OtherDepositTypeDesc)
+                .Map(dest => dest.PersonDepositHolder, src => src.PimsSecurityDepositHolders.FirstOrDefault(h => h.Person != null))
+                .Map(dest => dest.PersonDepositHolderId, src => src.PimsSecurityDepositHolders.Select(h => h.PersonId).FirstOrDefault())
+                .Map(dest => dest.OrganizationDepositHolder, src => src.PimsSecurityDepositHolders.FirstOrDefault(h => h.Organization != null))
+                .Map(dest => dest.OrganizationDepositHolderId, src => src.PimsSecurityDepositHolders.Select(h => h.OrganizationId).FirstOrDefault())
```

Comment:
Also, when selecting Organizations you are making sure to select only non-null values. Should there not be similar logic for the organization id? otherwise the organization you select may not be the same as the organization id that gets selected.

Reply:
ie: this case: [{organization: null, organizationId: 1}, {organization: {blah} organizationId: 2}]
I think this logic will get an organizationId of 1 but return organization blah

Reply:
Recommend writing a function that both can use to get the organization, and then for the id you can call getOrganization().id

Reply:
This is similar to the previous comment. I think we will have to chat about the db model to make sure it matches the requirements before making further changes to this.

Reply:
not sure I agree on this one, regardless of whether mapping a many-to-one is correct, that mapping should return the same value consistently. You should not be returning the id from objectA while also returning objectB.

Reply:
I dont disagree with that. My point is that since those methods/members are related to the holder model is better if we wait until the model gets sorted out.